arch/arm/boot/dts/overlays/goodix-overlay.dts

```diff
@@ -0,0 +1,38 @@
+// Device tree overlay for I2C connected Goodix gt9271 multiple touch controller on Artista-IoT touch connector
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
+	fragment@0 {
+		target = <&gpio>;
+		__overlay__ {
+			goodix_pins: goodix_pins {
+				brcm,pins = <34 35>; // interrupt and reset on i2c touch connector
```

Comment:
You must be on Compute Module. Could make the pin usage parameters so it is usable by others? See [the gpiopin parameter in the w1-gpio overlay](https://github.com/raspberrypi/linux/blob/rpi-4.9.y/arch/arm/boot/dts/overlays/w1-gpio-overlay.dts#L35) for an example.  You would need two parameters - `interrupt` and `reset`, say. `interrupt` would need to override `<&gpio>,"brcm,pins:0",
 <&gt9271>,"interrupts:0", <&gt9271>,"irq-gpios:4";`. You can work out the other by analogy.

You can leave your values as defaults.